### PR TITLE
Use Renovate instead of Dependabot for npm udpates

### PR DESCRIPTION
**Pull request by DerekRoberts**
Renovate is in place and more or less running even with Dependabot.  Comment out npm packages from .github/dependabot.yml, leaving Dependabot only running GitHub Actions updates.

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-quickstart-typescript-878-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-quickstart-typescript-878-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-quickstart-typescript/actions/workflows/merge-main.yml)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,18 +7,18 @@ updates:
...
       interval: daily
 
-  - package-ecosystem: npm
-    directory: /backend
-    schedule:
-      interval: daily
-    commit-message:
-      prefix: "Backend"
-      include: "scope"
+  # - package-ecosystem: npm
```
Comment (paulushcgcj) [inline .github/dependabot.yml:10]: Isn't removing a better option? Even if it's just for testing purposes, we still have git to check the older values

**Review by paulushcgcj — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (DerekRoberts) [reply .github/dependabot.yml]: Sure!  Now it's removed.

**Review by paulushcgcj — APPROVED** [review]
Chef's Kiss

**Review by mishraomp — APPROVED** [review]
LGTM

**Pull request merged**